pkg/util/fs_util_test.go

```diff
@@ -694,3 +694,49 @@ func Test_childDirInWhitelist(t *testing.T) {
 		})
 	}
 }
+
+func Test_correctDockerignoreFileIsUsed(t *testing.T) {
+	type args struct {
+		dockerfilepath string
+		buildcontext   string
+		excluded       string
+		notExcluded    string
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		{
+			name: "relative dockerfile used",
+			args: args{
+				dockerfilepath: "../../integration/dockerfiles/Dockerfile_test_dockerignore_relative",
+				buildcontext:   "../../integration/",
+				excluded:       "ignore_relative/bar",
+				notExcluded:    "ignore_relative/foo",
+			},
+		},
+		{
+			name: "context dockerfile is used",
+			args: args{
+				dockerfilepath: "../../integration/dockerfiles/Dockerfile_test_dockerignore",
+				buildcontext:   "../../integration/",
+				excluded:       "ignore/bar",
+				notExcluded:    "ignore/foo",
```

Comment:
We should test `ignore_relative/bar` is not excluded here to make sure `Dockerfile_test_dockerignore_relative.dockerignore` is not used?